src/Raven.Studio/wwwroot/Content/css/pages/query.less

```diff
@@ -390,3 +390,15 @@
         }
     }
 }
+
+.tabs-menu {
```

Comment:
can we nest those inside .query or #query ?

Reply:
moved to be under `.query-results-container`